.github/workflows/issues.yml

```
@@ -0,0 +1,31 @@
+---
+name: GH issues
+on:
+  workflow_call:
+    inputs:
+      ado_area_path:
+        description: 'ADO area path where ADO tickets should be created'
+        required: true
```

```suggestion
        required: false
```

This should not be required in cases where `do-gh-to-ado` is false